templates/admin/confirm/overview.html

```diff
@@ -32,11 +33,12 @@
             <td v-else>
                 Bekräftad
             </td>
-            <td><a :href="'/expenses/' + expense.id + '/'" v-text="expense.id"></a></td>
+            <td><a :href="'/expenses/' + expense.id + '/'" v-text="expense.id"></a> </td>
             <td v-if="!(digital || analog)"><p v-if="expense.is_digital">Ja</p><p v-else-if="expense.is_digital == false">Nej</p><p v-else>👻</p></td>
             <td><a :href="'/expenses/' + expense.id + '/'" v-text="expense.description"></a></td>
             <td><a :href="'/users/' + expense.owner_username + '/'" v-text="expense.owner_first_name + ' ' + expense.owner_last_name"></a></td>
             <td v-text="expense.amount + ' kr'"></td>
+            <td><span class="icon"><i class="fa fa-flag" v-if="expense.is_flagged === false"></i></span></td>
```

Comment:
shouldn't it be
```suggestion
            <td><span class="icon"><i class="fa fa-flag" v-if="expense.is_flagged === true"></i></span></td>
```
?

Reply:
ok maybe not because this isn't red, but then what's the point? why only show this (unclickable) icon only when something has been unflagged? just for post-fix visibility?